lib/PuppeteerSharp.Tests/PuppeteerTests/PuppeteerLaunchTests.cs

```diff
@@ -291,12 +292,14 @@ public void ShouldDumpBrowserProcessStderr()
             var process = new Process();
 
 #if NETCOREAPP
-            process.StartInfo.WorkingDirectory = GetDumpIOAppDirectory();
+            process.StartInfo.WorkingDirectory = GetSubprocessWorkingDir("PuppeteerSharp.Tests.DumpIO");
```

Comment:
maybe create a helper function that will return that process? it should also contain the `#if NETCOREAPP` .
something like:
```cs
private Process  CreateProcess(string project)
{
    var process = new Process();
    process.StartInfo.WorkingDirectory = GetSubprocessWorkingDir(project);
    process.StartInfo.Arguments = $"{project}.dll {dumpioTextToLog} "
        + $"\"{new BrowserFetcher().RevisionInfo(BrowserFetcher.DefaultRevision).ExecutablePath}\"";
    return process;
}
```